app/workers/downstream_draft_worker.rb

```diff
@@ -54,7 +54,7 @@ def assign_attributes(attributes)
     @content_id = attributes.fetch(:content_id)
     @locale = attributes.fetch(:locale)
     @edition = Queries::GetEditionForContentStore.(content_id, locale, true)
-    @payload_version = attributes.fetch(:payload_version)
+    @payload_version = Event.where(content_id: content_id).maximum(:id) || 0
```

Comment:
Can we use the `Event.payload_version` method here?

Reply:
Whoops - thanks!